packages/next/server/render.tsx

```diff
@@ -208,13 +208,8 @@ function renderPageTree(
   isServerComponent: boolean
 ) {
   const { router: _, ...rest } = props
-
   if (isServerComponent) {
-    return (
-      <App>
-        <Component {...rest} />
-      </App>
-    )
+    return <Component {...rest} />
```

Comment:
I think we should keep `App` here but remove it from `ServerComponentWrapper`, _app shouldn't be rendered as a part of server components. What's your take?

Reply:
we have to keep it inside `ServerComponentWrapper` for now since `AppServer` needs to be sent through streaming response as well

Reply:
> since AppServer needs to be sent through streaming response as well

The definition of `_app.server` is blurry. If it is a client component, it shouldn't be included in the flight response. If it's a server component, it should be allowed to exist at the same time along with `_app`.

Reply:
_app is the client one, which is not included in flight response yet. and you can have _app.server and _app at the same time.